composer.json

```diff
@@ -10,14 +10,14 @@
   },
   "require": {
     "php": ">=7.3",
-    "illuminate/http": "^8.0 || ^9.0",
+    "illuminate/http": "^9.0 || ^10.0",
     "nyholm/psr7": "^1.2",
     "psr/http-server-middleware": "^1.0",
     "symfony/psr-http-message-bridge": "^2.0"
   },
   "require-dev": {
-    "friendsofphp/php-cs-fixer": "^2.16",
-    "phpunit/phpunit": "^9.0"
+    "friendsofphp/php-cs-fixer": "^3.0",
```

Comment:
When running: `docker-compose run fixcs`

I'm getting:
```
Fatal error: Uncaught Error: Call to undefined method PhpCsFixer\Config::create() in /app/.php-cs-fixer.php:6
```

Could you check it?